_posts/2021-09-21-tweet-hashtags-relations.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Twitter Hashtag Clustering Network: Visualizing Popular Twitter Hashtags"
+usemathjax: false
+image: /public/images/HRprofilepic.png
+summary: This project showcases visualizing a dataset containing categorical and numerical data, and also build a pipeline that deals with missing data, imbalanced data and predicts a binary outcome. The original dataset can be found on Kaggle, and full details including all of my code is available in a notebook on Kaggle.
+---
+As life getting faster paced, hashtags and tweets are getting more popular on social media. This is perhaps caused by the conciseness of hashtags when delivering messages and information, which served the fast paced life well. 
+
+It is not very hard to notice hashtags often co-occur in the same tweet. If you click on one hashtag on Twitter, a page with all the tweets containing the clicked hashtag and co-occurred hashtags would appear. This means the clicked hashtag can lead to many other hashtags. In this project, I will analyze and visualize the probability of one hashtag leading to another hashtag. 
```

Comment:
change to:
containing the clicked hashtag would appear, along with many co-occurred hashtags.